### PR TITLE
Monte carlo implementation in scala

### PR DESCRIPTION
**Pull request by kenpower**

### CODE REVIEW

File: contents/monte_carlo_integration/monte_carlo_integration.md [hunk 1]
```diff
@@ -71,6 +71,8 @@ each point is tested to see whether it's in the circle or not:
 [import:1-4, lang:"ruby"](code/ruby/monte_carlo.rb)
 {% sample lang="f90" %}
 [import:1-8, lang:"fortran"](code/fortran/monte_carlo.f90)
+{% sample lang="scala" %}
+[import:3-14, lang:"scala"](code/scala/monte_carlo.scala)
```
Comment (jiegillet) [inline contents/monte_carlo_integration/monte_carlo_integration.md]: Oh noooo! `import:3-14` would have been perfect for this algorithm, unfortunately here you only need to import line 3 :(

**Comment by jiegillet** [comment]
Also, you should really use `git rebase -i upstream master` before pushing your branch because your commit history is very hard to follow. Another trick I've had to clean up my master branch use was `git reset --hard upstream/master`

File: contents/monte_carlo_integration/monte_carlo_integration.md (hunk 1 above)
Comment (jiegillet) [reply contents/monte_carlo_integration/monte_carlo_integration.md]: That, or maybe `import:3-3` I'm not sure. You can try both and then compile the book for yourself and see.

**Comment by berquist** [comment]
As long as it isn't merged, it's not too late. Once you enter interactive rebase, "fixup" all except the first commit to nuke their commit messages ("squash" doesn't do this), save/commit, force push, then you're good.

**Comment by jiegillet** [comment]
Also we are always squishing the commits on GitHub before merging PRs so it doesn't really matter for the end result, it's just easier to follow for the reviewers. And now it's back to one commit so it isn't necessarily easy to read either :)
